harness/setup.py

```diff
@@ -1,66 +1,42 @@
+import os
+import subprocess
+
 import setuptools
 
-# open README.md from parent folder and read it into a string
-with open("../README.md", "r") as readme:
-    markdown_description = "".join(readme.readlines())
+
+def readme() -> str:
+    with open("../README.md", "r") as fd:
+        return fd.read()
+
+
+def version() -> str:
+    version = os.environ.get("VERSION")
+
+    if version is None:
```

Comment:
put all this in `get_version_from_sh`, keep it scoped inside this function

Reply:
Good catch, thanks!